community/kernel/src/test/java/org/neo4j/kernel/lifecycle/LifeRule.java

```diff
@@ -52,12 +53,13 @@ public void evaluate() throws Throwable
                         start();
                     }
                     base.evaluate();
+                    life.shutdown();
```

Comment:
We used `start()` method before `base.evaluate()`. Can we change `life.shutdown()` to `shutdown()` here and in catch block?